meshtastic/interfaces/ble/interface.py

```
@@ -0,0 +1,1899 @@
+                    return_adv=True,
+                    service_uuids=[SERVICE_UUID],
+                )
+                return _parse_scan_response(response)
+            except BleakDBusError:
+                # Propagate DBus-level failures so callers can back off appropriately
+                raise
+            except (BleakError, RuntimeError) as e:
+                logger.warning("Device scan failed: %s", e, exc_info=True)
+                return []
+            except (
+                Exception
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)